### DISCUSSION TITLE
Multiple entitlement option for Fastlane resign

### DISCUSSION THREAD

**Discussion by bobrenji** [fastlane/fastlane#19817] [Q&A]
I am building an app with extensions. I have enabled capabilities like keychain sharing, associated-domains etc.

So I have to use resign action with custom entitlements. Problem for me, When I resign my app and try to upload App Store. It fails with error. I can see both my app and extension target used same app-id entitlement. Is there any way I can provide multiple entitlements. So I can handle it properly.

So any idea?

**Comment by CodeWithOz** [comment]
Have you found a solution to this yet? Fastlane allows `match`ing multiple targets (i.e. bundle IDs) but I'm not sure how to sign them with their respective profiles after `match` completes.

**Comment by Carissa34** [comment]
Automatically renew your provisioning profiles to include all your devices using the --force option., Support for multiple Apple accounts and multiple 

https://www.dgcustomerfirst.win/

**Comment by CodeWithOz** [comment]
Sorry I'm not clear on what you mean. My profiles are already renewed and managed using `match`. Where are you applying the `--force` option?
